CMakeLists.txt

```diff
@@ -99,10 +107,26 @@ endif()
 add_library(LibDataChannel::LibDataChannel ALIAS datachannel)
 add_library(LibDataChannel::LibDataChannelStatic ALIAS datachannel-static)
 
+# Main Test
 add_executable(tests ${TESTS_SOURCES})
 set_target_properties(tests PROPERTIES
 	VERSION ${PROJECT_VERSION}
 	CXX_STANDARD 17)
 
 target_link_libraries(tests datachannel)
 
+# P2P Test: Peer-1
+add_executable(peer-1 ${TESTS_PEER_1_SOURCES})
```

Comment:
Why not renaming `peer-1` and `peer-2` as `offerer` and `answerer`? I think it would make things clearer for the user.